README.md

```diff
@@ -12,10 +21,18 @@ Now you can reference the live value of *x* in any cell, *e.g.*:
 md`The value of *x* is ${x}.`
 ```
 
-Any cell that references *x* will run automatically when the *viewof x* slider is moved. For live examples, see:
+Any cell that references *x* will run automatically when the slider is moved. For live examples, see:
 
 https://observablehq.com/@observablehq/inputs
 
+In [Observable Framework](https://observablehq/com/framework), the reactivity is ensured with the *view* function, that both displays the input and returns the value generator. Write:
```

Comment:
The URL is broken here: `observablehq/com` instead of ``observablehq.com`. 🤦 I’ll fix… 😓

Reply:
oops that was a bad day!